src/test/java/core/basesyntax/service/CsvFruitTransactionParserImplTest.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class CsvFruitTransactionParserImplTest {
+    private FruitTransactionParser parser = new CsvFruitTransactionParserImpl();
```

Comment:
services better initialize in methods annotated with before or beforeclass (depends or the unit version)